packages/apollo-server-express/src/__tests__/ApolloServer.test.ts

```diff
@@ -184,13 +184,12 @@ describe('apollo-server-express', () => {
     });
 
     it('renders GraphQL playground using request original url', async () => {
-      const nodeEnv = process.env.NODE_ENV;
-      delete process.env.NODE_ENV;
       const samplePath = '/innerSamplePath';
 
       const rewiredServer = new ApolloServer({
         typeDefs,
         resolvers,
+        playground: true,
```

Comment:
Was this just a hacky workaround to make playground render in a testing env?

Reply:
I'm thinking about this as the opposite — setting NODE_ENV to mean "render playground" is a hacky workaround, whereas given that this is a test of playground functionality, asking for what you need makes sense. I left in some tests which set NODE_ENV which are explicitly saying "make sure playground is on by default in production" though.

Reply:
I think we're on the same page. This _was_ a hacky workaround, but I prefer the change you made here. Just confirming my understanding of the change.